server/app/views/applicant/ApplicantCommonIntakeUpsellTemplate.html

```diff
@@ -0,0 +1,57 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+    <main role="main">
+      <h1
```

Comment:
you could probably consolidate these two tags and make it:

`<h1 th:text="${isTrustedIntermediary ? #{title.commonIntakeConfirmationTi} : #{title.commonIntakeConfirmation}}"></h1>`

i haven't tested that so it might not be 100% right but something should be possible there

Reply:
same as elsewhere in the file